app/src/components/page/Sidebar/index.tsx

```diff
@@ -78,7 +78,7 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
     <Layout.Sider width="275px" className="layout-sider">
       <div className="logo">
         <Link to={URL_HOME}>
-          <img src={Logo} className="img-fluid" alt="" />
+          <img src={MAIN_LOGO_SRC} alt="" />
```

Comment:
`className="img-fluid"` is not needed any longer?

Reply:
I think i removed while playing around with some different styling, will add it back